### ISSUE TITLE
travis-ci for tests.

### ISSUE THREAD

**Issue by vincentdavis** [RyanRiddle/elgamal#5]
If you can setup travis-ci account, I will configure the .travis.yml file. This will also help if you really want to add python2 support.

**Comment by RyanRiddle** [comment]
Yeah that sounds great.  I added a .travis.yml file but I'm having trouble.  I'm getting this error 
"Can not perform a '--user' install. User site-packages are not visible in this virtualenv."
when running this command 
$ pip install --user -r requirements.txt
during the install step

**Comment by RyanRiddle** [comment]
Ok I got it working.  What do you think is the best way to use travis-ci to add python2 support?

**Comment by vincentdavis** [comment]
I sue pycharm as an editor, it has some feature for writing python2/3 code.
I have never used them but will take a look. Regarding how to use Travis in
the process. If your current tests just use the math then it would probably
be easy if they don't already work in python 2. I think testing python 2
and 3 maybe also pypy now and as tests are added make sure they pass.
